packages/api/AlvTimeWebApi/Controllers/FlexiHourController.cs

```diff
@@ -87,13 +87,9 @@ public ActionResult<PayoutsDto> FetchPaidOvertime()
         [Authorize]
         public ActionResult<GenericHourEntry> RegisterPaidOvertime([FromBody] GenericHourEntry request)
         {
-            if (request.Hours < 0)
+            if (!ModelState.IsValid)

```

Comment:
Du kan også tagge controllerne med [ApiController] for å koble automatisk håndtering av model state: https://www.strathweb.com/2018/02/exploring-the-apicontrollerattribute-and-its-features-for-asp-net-core-mvc-2-1/